validator/enforce_labels_test.rego

```diff
@@ -29,8 +29,8 @@ import data.test.fixtures.enforce_labels.assets.compute.instances as fixture_com
 import data.test.fixtures.enforce_labels.assets.compute.snapshots as fixture_compute_snapshots
 import data.test.fixtures.enforce_labels.assets.dataproc.clusters as fixture_dataproc_clusters
 import data.test.fixtures.enforce_labels.assets.dataproc.jobs as fixture_dataproc_jobs
-import data.test.fixtures.enforce_labels.assets.projects as fixture_projects
 import data.test.fixtures.enforce_labels.assets.gke as fixture_gke
+import data.test.fixtures.enforce_labels.assets.projects as fixture_projects
```

Comment:
I don't think you intended to update this file in this PR - can you merge master into your branch to make sure only your files are in this PR?

Reply:
This file is still in this PR